kong/runloop/plugin_servers/process.lua

```diff
@@ -235,8 +276,20 @@ function proc_mgmt.pluginserver_timer(premature, server_def)
     end
   end
   kong.log.notice("Exiting: pluginserver '", server_def.name, "' not respawned.")
+
 end
 
+-- limit the number of warning messages
+local plugin_already_warned = {}
+function proc_mgmt.handle_not_ready(plugin_name)
+  if plugin_already_warned[plugin_name] then
+    return
+  end
+
+  plugin_already_warned[plugin_name] = true
+  log(ngx_WARN, "plugin server is not ready when ",
+      plugin_name, " is required for request")
```

Comment:
```suggestion
  log(ngx_WARN, "plugin server ", plugin_name,
      " is not started, plugin on this plugin server will",
      " not be executed")
```

Reply:
`plugin_name` is plugin's name, not the plugin server's name